py/circuitpy_mpconfig.mk

```diff
@@ -555,6 +555,9 @@ CFLAGS += -DCIRCUITPY_TERMINALIO=$(CIRCUITPY_TERMINALIO)
 CIRCUITPY_FONTIO ?= $(call enable-if-all,$(CIRCUITPY_DISPLAYIO) $(CIRCUITPY_TERMINALIO))
 CFLAGS += -DCIRCUITPY_FONTIO=$(CIRCUITPY_FONTIO)
 
+CIRCUITPY_TILEPALETTEMAPPER ?= $(CIRCUITPY_DISPLAYIO)
+CFLAGS += -DCIRCUITPY_TILEPALETTEMAPPER=$(CIRCUITPY_TILEPALETTEMAPPER)
+
```

Comment:
Rather than setting two parameters in the atmel-samd/boards mpconfigboard.mk files, I think the CIRCUITPY_TERMINALIO_VT100 I recently created should default to your new parameter. (After #10108 was merged, it's being set immediately following CIRCUITPY_TERMINALIO, but doesn't show up here?). I'm also setting the VT100 parameter to 0 for all samd21 boards in the atmel-samd mpconfigport.mk which could be removed if you make this change and decide to disable the new palette for all samd21 boards :grin:

Reply:
As I think about it, the two features aren't technically, necessarily linked so perhaps two parameters is appropriate. Trade-offs... :shrug:

Reply:
I think keeping them separate makes sense, but I also don't have a strong preference one way or the other. If it's possible to set it in one spot as disabled for all SAMD21 boards that sounds more convenient than adding to each of board individually though.

Reply:
Putting your assignment here will turn it off for all samd21 boards by default. The board files can still turn it back on if needed.

https://github.com/adafruit/circuitpython/blob/e8de36a9f8a2a27a77a1888a6db389c548f426e3/ports/atmel-samd/mpconfigport.mk#L60-L62

Reply:
Thank you! I do want to keep it on for any samd51s that have room. The pyportal titano for instance is what all my testing was on and it seems fine on that device.